src/Panels/ConnectionPanel.cs

```diff
@@ -157,7 +182,25 @@ public override void Start()
                 isVisible = false;
             };
 
-            base.Start();
```

Comment:
Superclass should also be called or why did you remove?

Reply:
As of now I'm on my phone so I'll see the biggest changes at home but I'll close this and restart the entire thing over again. This is becoming more complicated to try separate different commits for certain pull requests. #donotmultitask